### DISCUSSION TITLE
page-navigation: is there a way to hide it in a specific page?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#8037] [Q&A] [answered]
### Description

Hi,
I would like to hide the page-navigation in the home of a quarto HTML book, because the home is not part of a book, it is just an introduction.

Can I hide it? How?

Thank you

**Comment by mcanouil** [comment, marked as answer]
Did you try setting `page-navigation: false` on the page you don't want it using 1.4 pre-release?

**Reply by aborruso** [reply]
> Did you try setting page-navigation: false on the page you don't want it using 1.4 pre-release?

No, I had not found any hint in the guide that it could also be applied per page. I was wrong.

It works fine, thank you

**Reply by mcanouil** [reply]
It might not be documented yet but it is in the changelog of 1.4: <https://quarto.org/docs/download/>
